### DISCUSSION TITLE
Taskbar badge notifications for Windows apps

### DISCUSSION THREAD

**Discussion by tedchirvasiu** [dotnet/maui#22768] [Ideas]
### Description

I'd love for MAUI to have working Badge Notifications as we used to have in UWP.

https://learn.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/badges

Currently trying to display a glyph in the taskbar icon just fails silently (although it is being displayed in the Start Menu). It has more the vibe of a bug than a missing feature to be honest, especially as the silent fail leads to hair pulling and possibly hours of digging to find out why.

Possibly related to https://github.com/microsoft/microsoft-ui-xaml/issues/1919 ?

Is there any other way to achieve this in MAUI or is it simply something we'd need to live without?

### Public API Changes

Getting the UWP API to work properly would suffice.

### Intended Use-Case

I'd love to have a way to display a busy glyph badge on the app icon to indicate it is doing a certain task, similar to OBS Studio when it is recording. This helps users from having to constantly switch to the app to check if it is running a certain task. In case of certain apps (such as screen recorders, timers, etc.) this feature is extremely important as it helps avoid users forgetting to do certain important actions (start recording, start the timer etc.).

![image](https://user-images.githubusercontent.com/13540243/225700206-114ed0f7-b196-4b52-a825-00f5815bca67.png)

**Comment by ghost** [comment]
We've added this issue to our backlog, and we will work to address it as time and resources allow. If you have any additional information or questions about this issue, please leave a comment. For additional info about issue management, please read our [Triage Process](https://github.com/dotnet/maui/blob/main/docs/TriageProcess.md).
